daemon/daemon.go

```diff
@@ -680,14 +693,17 @@ func (d *daemon) ResetExercise(req *pb.ResetExerciseRequest, stream pb.Daemon_Re
 	for _, t := range ev.GetTeams() {
 		lab, ok := ev.GetLabByTeam(t.Id)
 		if !ok {
-			stream.Send(&pb.ResetTeamStatus{TeamId: t.Id, Status: "?"})
+			if err:=stream.Send(&pb.ResetTeamStatus{TeamId: t.Id, Status: "?"}); err!=nil {
```

Comment:
Hmm is it just me or did gofmt overlook this one.

Reply:
I think, gofmt did not applied :(